flexget/plugins/sites/site1337x.py

```diff
@@ -11,10 +11,12 @@
 from flexget.utils.requests import RequestException
```

Comment:
paranoidi changed the class name to remove the underscore as it wasn't pep8.  I changed the name of the file to match the class name